### PR TITLE
Replace some feedback categories

### PR DESCRIPTION
**Pull request by frederoni**
Replaced some of the feedback categories.

7+ and 7.

@ericrwolfe @bsudekum  👀

### CODE REVIEW

File: MapboxNavigation/FeedbackViewController.swift [hunk 1]
```diff
@@ -33,31 +33,31 @@ class FeedbackViewController: UIViewController {
...
-        let confusingTitle      = NSLocalizedString("FEEDBACK_CONFUSING", bundle: .mapboxNavigation, value: "Confusing", comment: "Feedback type for Confusing")
-        let otherIssueTitle     = NSLocalizedString("FEEDBACK_OTHER", bundle: .mapboxNavigation, value: "Other Issue", comment: "Feedback type for Other Issue")
+        let instructionTimingTitle      = NSLocalizedString("FEEDBACK_INSTRUCTION_TIMING", bundle: .mapboxNavigation, value: "Instruction \nTiming", comment: "Feedback type for Instruction Timing")
+        let confusingInstructionTitle   = NSLocalizedString("FEEDBACK_CONFUSING_INSTRUCTION", bundle: .mapboxNavigation, value: "Confusing \nInstruction", comment: "Feedback type for Confusing Instruction")
+        let notAllowedTitle             = NSLocalizedString("FEEDBACK_NOT_ALLOWED", bundle: .mapboxNavigation, value: "Not \nAllowed", comment: "Feedback type for turn not allowed")
+        let gpsInaccurateTitle          = NSLocalizedString("FEEDBACK_GPS_INACCURATE", bundle: .mapboxNavigation, value: "GPS \nInaccurate", comment: "Feedback type for inaccurate GPS")
```
Comment (1ec5) [inline MapboxNavigation/FeedbackViewController.swift]: Be sure to update Navigation.strings so that the new categories can be localized.

Comment (frederoni) [reply MapboxNavigation/FeedbackViewController.swift]: We're moving this to a remote source asap so translating all these strings here felt like an excessive step.

**Comment by bsudekum** [comment]
Do we need new icons before merging this?

**Comment by ericrwolfe** [comment]
> Do we need new icons before merging this?

Yes. Let's hold on merging this until we have the updated UI and icons this week.

**Comment by bsudekum** [comment]
Can we close this out? https://github.com/mapbox/mapbox-navigation-ios/pull/739 took care of this.

**Pull request closed without merging**